xclim/indices/_conversion.py

```diff
@@ -1818,6 +1819,9 @@ def universal_thermal_climate_index(
         If True (default), UTCI values are NaN where any of the inputs are outside
         their validity ranges : -50°C < tas < 50°C,  -30°C < tas - mrt < 30°C
         and  0.5 m/s < sfcWind < 17.0 m/s.
+    wind_cap_min: bool
+        If True, low wind velocities are capped to a minimum of 0.5 m/s. This ensures
+        UTCI calculation for low winds. Default value False.
```

Comment:
Oh, would it be interesting to cite Bröde 2012 here ? Or below in the "references" section ?
In any case, the metadata for the publication has to be added to "docs/references.bib" before being cited here with `:cite:cts:` (for a long version) or `:cite:t:` (short).

Reply:
I think is better to cite Bröde 2012 (and also Bröde 2023 Figure 2) here because the user may want to know a bit more context of this specific option. I will include both in the "docs/reference.bib" file and update the description